### ISSUE TITLE
Cannot run codium 1.39.1 : Failed to connect to dbus/system_bus_socket

### ISSUE THREAD

**Issue by vincowl** [skycocker/chromebrew#3582]
Running `codium` on x86_64 architecture (asus flip c302ca), codium doesn't start, no error is shown.
While running it in verbose mode on a fresh chromebrew install :
```shell
$ codium --verbose
[11268:1026/145132.063378:ERROR:bus.cc(394)] Failed to connect to the bus: Failed to connect to socket /usr/local/var/run/dbus/system_bus_socket: No such file or directory
[11298:1026/145132.159156:ERROR:gl_surface_glx.cc(424)] glxQueryVersion failed
[11298:1026/145132.165428:ERROR:gl_initializer_x11.cc(157)] GLSurfaceGLX::InitializeOneOff failed.
[11298:1026/145132.183312:ERROR:viz_main_impl.cc(236)] Exiting GPU process due to errors during initialization
[main 2019-10-26T12:51:32.372Z] Starting VS Code
[main 2019-10-26T12:51:32.373Z] from: /usr/local/share/VSCodium-linux-x64/resources/app
[main 2019-10-26T12:51:32.373Z] args: { _: [], verbose: true, diff: undefined }
[main 2019-10-26T12:51:32.376Z] Resolving machine identifier...
[main 2019-10-26T12:51:32.376Z] Resolved machine identifier: 3b77775d-4557-4149-9b41-ca1a45b61eed (trueMachineId: undefined)
[main 2019-10-26T12:51:32.422Z] update#setState idle
[main 2019-10-26T12:51:32.428Z] [storage state.vscdb] open(/home/chronos/user/.config/VSCodium/User/globalStorage/state.vscdb, retryOnBusy: true)
[main 2019-10-26T12:51:32.429Z] lifecycle (main): phase changed (value: 2)
[main 2019-10-26T12:51:32.432Z] windowsManager#open
```
I tried to run other graphical apps such as filezilla or gimp which are correctly displayed.

**Comment by vincowl** [comment]
Here are a few infos on my dbus status
```shell
$ dbus-uuidgen --ensure
$ /usr/bin/dbus-daemon --system
/usr/bin/dbus-daemon: /usr/local/lib64/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.10.12' not found (required by /usr/bin/dbus-daemon)
$ /usr/bin/dbus-daemon --session
/usr/bin/dbus-daemon: /usr/local/lib64/libdbus-1.so.3: version `LIBDBUS_PRIVATE_1.10.12' not found (required by /usr/bin/dbus-daemon)
$ dbus-launch --sh-syntax
DBUS_SESSION_BUS_ADDRESS='unix:abstract=/tmp/dbus-3y7eNrlPd6,guid=89c95b026ac882442c0cd8815db4441c';
export DBUS_SESSION_BUS_ADDRESS;
DBUS_SESSION_BUS_PID=26619;
DBUS_SESSION_BUS_WINDOWID=4194305;
```

**Comment by uberhacker** [comment]
@vincowl:  Thank you for the feedback.  Did you install the dbus package?  Can you please post the output of `cat /etc/chromeos-release`?

**Comment by uberhacker** [comment]
There is an update available also: https://github.com/VSCodium/vscodium/releases/download/1.39.2/VSCodium-linux-x64-1.39.2.tar.gz

**Comment by vincowl** [comment]
@uberhacker Yes, I installed dbus and dbus_glib. 
```
$ cat /etc/lsb-release
CHROMEOS_ARC_ANDROID_SDK_VERSION=28
CHROMEOS_ARC_VERSION=5896756
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={B00DD0BC-D2C9-BAB4-E66C-81AE3F5A7CED}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={B00DD0BC-D2C9-BAB4-E66C-81AE3F5A7CED}
CHROMEOS_RELEASE_BOARD=cave-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=75
CHROMEOS_RELEASE_BUILDER_PATH=cave-release/R77-12371.75.0
CHROMEOS_RELEASE_BUILD_NUMBER=12371
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=77
CHROMEOS_RELEASE_DESCRIPTION=12371.75.0 (Official Build) stable-channel cave 
CHROMEOS_RELEASE_KEYSET=mp
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=12371.75.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=12371.75.0
```

**Comment by vincowl** [comment]
Same error with 1.39.2 release. It seems to be related to dbus itself since basic dbus start command are failing

**Comment by uberhacker** [comment]
I was able to run codium fine.  Did you make sure sommelier was running?

**Comment by uberhacker** [comment]
FWIW: I get the same error as you but it doesn't prevent codium from launching.
```
[32674:1026/170256.943742:ERROR:bus.cc(394)] Failed to connect to the bus: Failed to connect to socket /usr/local/var/run/dbus/system_bus_socket: No such file or directory
[32701:1026/170257.098223:ERROR:gl_surface_glx.cc(424)] glxQueryVersion failed
[32701:1026/170257.098293:ERROR:gl_initializer_x11.cc(157)] GLSurfaceGLX::InitializeOneOff failed.
[32701:1026/170257.123446:ERROR:viz_main_impl.cc(236)] Exiting GPU process due to errors during initialization
[main 2019-10-26T22:02:57.331Z] Starting VS Code
[main 2019-10-26T22:02:57.338Z] from: /usr/local/share/VSCodium-linux-x64/resources/app
[main 2019-10-26T22:02:57.339Z] args: { _: [], verbose: true, diff: undefined }
[main 2019-10-26T22:02:57.348Z] Resolving machine identifier...
[main 2019-10-26T22:02:57.349Z] Resolved machine identifier: cb5bf631-e02c-4719-9377-d45930e6f7d5 (trueMachineId: undefined)
[main 2019-10-26T22:02:57.426Z] update#setState idle
```

**Comment by vincowl** [comment]
Startsommelier is part of my .bashrc. I ran gimp and filezilla without trouble. I suppose sommelier was correctly running. That's weird... I will rebuild a fresh chromebrew install tomorrow and check

**Comment by vincowl** [comment]
Found what seems to create the problem : with a fresh install codium works perfeclty, but as soon as I install pulseaudio (php dependancy), codium doesn't show up

**Comment by vincowl** [comment]
I finally solved this : install pulseaudio 13.0 then install codium.
